models/spleen_ct_segmentation/docs/README.md

```diff
@@ -44,6 +44,14 @@ Dice score is used for evaluating the performance of the model. This model achie
 #### Validation Dice
 ![A graph showing the validation mean Dice over 1260 epochs.](https://developer.download.nvidia.com/assets/Clara/Images/clara_pt_spleen_ct_segmentation_val_2.png)
 
+#### TensorRT speedup
+The `spleen_ct_segmentation` bundle supports the TensorRT acceleration. The table below shows the speedup ratios benchmarked on an A100 80G GPU, in which the `model computation` means the speedup ratio of model's inference with a random input without preprocessing and postprocessing and the `end2end` means run the bundle end to end with the TensorRT based model. The `torch_fp32` and `torch_amp` is for the pytorch model with or without `amp` mode. The `trt_fp32` and `trt_fp16` is for the TensorRT based model converted in corresponding precision. The `speedup amp`, `speedup fp32` and `speedup fp16` is the speedup ratio of corresponding models versus the pytorch float32 model, while the `amp vs fp16` is between the pytorch amp model and the TensorRT float16 based model.
+
+| method | torch_fp32(ms) | torch_amp(ms) | trt_fp32(ms) | trt_fp16(ms) | speedup amp | speedup fp32 | speedup fp16 | amp vs fp16|
+| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+| model computation | 6.48 | 4.48 | 6.40 | 6.30 | 1.45 | 1.01 | 1.03 | 0.71 |
```

Comment:
Should we use the ONNX workaround here?

Thanks.

Reply:
Ok, I can update this part.

Reply:
And where you plan to put the description for ONNX workaround?

Thanks.

Reply:
Since currently only the spleen ct segmentation bundle is using onnx workaround, I think we'd better just put it in the README file of spleen ct segmentation. How about this?

Reply:
But you will update several other bundles with ONNX workaround soon, right?
So I think maybe it's better to describe in the contribution guide directly?

Thanks.